pkg/cloud/services/loadbalancer/loadbalancer.go

```diff
@@ -56,7 +57,12 @@ func (s *Service) ReconcileLoadBalancer(openStackCluster *infrav1.OpenStackClust
 			VipSubnetID: openStackCluster.Status.Network.Subnet.ID,
 		}
 
+		mc := metrics.NewMetricPrometheusContext("server", "create")
+
 		lb, err = loadbalancers.Create(s.loadbalancerClient, lbCreateOpts).Extract()
+
+		mc.ObserveRequest(err)
+
 		if err != nil {
 			record.Warnf(openStackCluster, "FailedCreateLoadBalancer", "Failed to create load balancer %s: %v", loadBalancerName, err)
 			return err
```

Comment:
Do you think we could add a metric for the gophercloud caals in line 106, line 132, line 156, line 227, line 249, line 290, and line 344 in addition to all get/list requests?